Cargo.lock

```diff
@@ -3295,9 +3295,9 @@ dependencies = [
 
 [[package]]
 name = "foundry-block-explorers"
-version = "0.2.7"
+version = "0.2.8"
```

Comment:
this should've been 0.3 because breaking bumps propagate if exposed, e.g. compilers in block-explorers